gateway/ln-gateway/src/lnrpc_client.rs

```diff
@@ -99,6 +102,8 @@ pub trait ILnRpcClient: Debug + Send + Sync {
         &self,
         htlc: InterceptHtlcResponse,
     ) -> Result<EmptyResponse, LightningRpcError>;
+
+    async fn update_scids(&self, scids: Vec<u64>) -> Result<EmptyResponse, LightningRpcError>;
```

Comment:
Let's add a doc comment here too, we probably should add one for `complete_htlc` as well :)